styles.css

```diff
@@ -0,0 +1,190 @@
+/* style for body is temporary */
+body {
+  max-width: max-content;
+  margin: auto;
+  border: 20px solid black;
```

Comment:
[ ] kindly remove this body border 👀 , as isn't a part f the design.
the Figma design :

![figdesi](https://user-images.githubusercontent.com/62156607/192010830-6801ca74-76bd-4dce-826c-22bdd443e12d.PNG)

you're design : 

![herdes](https://user-images.githubusercontent.com/62156607/192010766-a7ab7719-5d12-48df-a736-67f1cb34ad75.PNG)

Reply:
I mentioned in the styles.css file, the border is temporary.